### ISSUE TITLE
Store sensor lengths in PuffinPlot data files

### ISSUE THREAD

**Issue by pont-us** [pont-us/PuffinPlot#341]
This can be done even if they're not included in the 2G file.

Doing it right will require some thought. I would expect that in 99% (at least) of cases, sensor length will be the same throughout the file, but of course 2G can save it per treatment step, and one could at least imagine composite long core files with sections measured on different magnetometers. In the worst case, one could even imagine that a core gets partially demagnetized on one magnetometer and some remaining steps are performed on another.

In conclusion, it may be necessary to write sensor lengths for every treatment step.

Note that this does *not* imply that magnetic moments should be stored without volume correction and corrected on the fly, but it does provide the possibility to check, undo, and/or redo the sensor length correction at any point, e.g. if it is suspected or confirmed that the wrong sensor lengths were initially set.
